Table implementation

git-user-cpp opened the pull request.

git-user-cpp committed:
Validation was implemented for some methods

git-user-cpp committed:
Validation was fixed

git-user-cpp committed:
update

git-user-cpp committed:
Method addNewRows was implemented

git-user-cpp committed:
Some bugs were fixed:
In class Table:
    - method verifyColumnNamesCorrect was re-written
In class TableRow:
    - method findDataObject was fixed

git-user-cpp committed:
Need to fix

git-user-cpp committed:
Need to fix CreateTableCommand

git-user-cpp committed:
Fix CreateTableCommand class

VladimirPetrat commented on MainModule/src/main/java/model/data/Table.java:
```diff
@@ -30,6 +30,19 @@ public String addNewRow(List<DataObject> dataPackage) {
     }
+    
+    public List<String> addNewRows(List<List<DataObject>> dataPackages) {
+        List<String> ids = new ArrayList<>();
```
Reuse addNewRow method

git-user-cpp replied:
done

git-user-cpp committed:
In class Table:
- method addNewRows was optimized

git-user-cpp reviewed (commented):
.

git-user-cpp committed:
Need to fix

git-user-cpp committed:
In class CreateTableCommand:
- method executeCommand was fixed
- the test was passed

git-user-cpp committed:
Code was formated due to Java standarts

git-user-cpp committed:
In class CreateTableCommand:
- method executeCommand was optimized

git-user-cpp committed:
Changes:
In Table class:
    - added getColumnNames()
In DataHandler class:
    - added returnTable()

git-user-cpp committed:
In class Table unnececary getter was removed

VladimirPetrat committed:
Reformated

VladimirPetrat committed:
Merge pull request #8 from VladimirPetrat/toTest

Reformated

git-user-cpp committed:
In class Table:
- default constructor was replaced by static factory method to improve validation during creating an object

git-user-cpp committed:
unnecessary brackets were removed
